### DISCUSSION TITLE
Question on Updating Alpha (in Natural Gas Supply Curve)

### DISCUSSION THREAD

**Discussion by davidbadler** [NREL/ReEDS-2.0#146] [Q&A: Data] [answered]
Hello,

We are trying to update some data in ReEDS with the recently released AEO2023. For the natural gas prices, we are having trouble updating the alpha parameters described in the Appendix of the model documentation. As a first step, we tried to match the alpha_AEO2022_reference.csv parameters using data from AEO2022. However, we are having difficulty getting our estimates to line up. We are using the NG demand and price data from AEO2022 within ReEDS2022, and I believe we found the beta's in input/scalars.csv and inputs/fuelprices/cd_beta0.csv (please let me know if these are incorrect!). Is there any sort of year restriction on the regression or use of say the prior year's demand/prices for calculating the subsequent year's alpha values?

In case it helps, a simplified example for checking the alpha parameters for a given region ("NE") in 2022 is attached as a reference. Is there anything that jumps out that we are missing/misunderstanding in the calculations of the alpha parameters?

Thanks very much in advance for your help!
[alpha_test_2022.xlsx](https://github.com/NREL/ReEDS_OpenAccess/files/11255570/alpha_test_2022.xlsx)

**Comment by wesleyjcole** [comment, marked as answer]
I recently made the update to AEO 2023 natural gas prices for an internal model version.  I uploaded the spreadsheet below that I used to make that price update.  The NG Alphas tab has the alphas that need to be ingested in the model.  It's the same one that I've used in prior years, so if you dropped in the appropriate AEO 2022 prices you should get an exact match in the final output sheet.

[NG Prices Preprocessing for AEO Inputs.xlsx](https://github.com/NREL/ReEDS_OpenAccess/files/11255756/NG.Prices.Preprocessing.for.AEO.Inputs.xlsx)

**Comment by davidbadler** [comment]
Thank you for the prompt response and for sharing the spreadsheet. This is perfect, thank you!
